### PR TITLE
Create AutomatedA11yTesting.md

### PR DESCRIPTION
**Pull request by j-paquette**
Migrating this doc:
 https://discourse.devx.team/t/automated-a11y-testing-recommendation/241/15

### CODE REVIEW

**Review by s-laugh — CHANGES_REQUESTED** [review]
I think you missed the main section of the recommendation (all the way at the top of the post). https://discourse.devx.team/t/automated-a11y-testing-recommendation/241
This is just a comment about some tools CDS has created.

File: Testing/AutomatedA11yTesting.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+There are quite a few free or open source a11y tools out there that we can use to automate the testing of our work.
+
+The developer experience team reached out to some folks in the web a11y community in Ottawa and the GoC through Twitter [Calvin Rodo's thread link](https://twitter.com/CalvinR/status/1093265325041352722) and were provided with some tools to look into.
+
+## Proof of Concept
+
+The DevX Team wrote a proof of concept (not completed) using Puppeteer and aXe Core.
+
+The link to this proof of concept is below: <https://github.com/esdc-devx/auto-a11y-demo>
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: Change to link formating
```suggestion
The link to this proof of concept is below: [https://github.com/esdc-devx/auto-a11y-demo](https://github.com/esdc-devx/auto-a11y-demo)
```

File: Testing/AutomatedA11yTesting.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Tools Selected
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: As it's part of the PoC lets add the header levels (not part of original document)
```suggestion
### Tools Selected
```

File: Testing/AutomatedA11yTesting.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+The DevX team was committed to selecting best in class open source tools for this proof of concept and we are fairly confident that we made the right choice. We ultimately ended up selecting [Puppeteer](https://developers.google.com/web/tools/puppeteer/) and [aXe Core](https://www.deque.com/axe/) for this demo.
+
+## Puppeteer
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: Once again based on the tools so lets add header levels.
```suggestion
#### Puppeteer
```

File: Testing/AutomatedA11yTesting.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Puppeteer is a NodeJS (JavaScript) Library for automating Google’s Chrome & Chromium Web Browser. The reason we choose this library over another tool such as selenium was purely due to ease of use. We found that Puppeteer was easier for our devs to use then other alternatives.
+
+## aXe Core
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: More header levels
```suggestion
#### aXe Core
```

File: Testing/AutomatedA11yTesting.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+aXe Core was selected due to it’s commitment to producing no-false positives and it is quickly becoming the go to library for other automation tools. For instance it’s used by Google for their in-browser and cli a11y automation tool Lighthouse. It also was very easy to integrate with Puppeteer.
+
+## Site Selected
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: More header levels (not part of tools)
```suggestion
### Site Selected
```

File: Testing/AutomatedA11yTesting.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+We decided to develop our proof of concept against the ROEWeb Demo site.
+
+The ROEWeb Demo Site is a site provided by ROEWeb that allows employers to train their staff and test out their tools against ROEWeb. The code base is identical to the production site, and more importantly it was designed to be testable and I (Calvin Rodo) spent several years working on it and so am fairly knowledgeable of the site.
+
+## Issues Encountered
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: More header levels
```suggestion
### Issues Encountered
```

File: Testing/AutomatedA11yTesting.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+<https://github.com/cds-snc/lighthouse-scanner>
+
+## cds-snc/loghthouse-scan-monitor
+
+<https://github.com/cds-snc/lighthouse-scan-monitor>
+
+## cds/snc/lighthouse-scanner-ui
+
+<https://github.com/cds-snc/lighthouse-scanner-ui>
```
Comment (s-laugh) [inline Testing/AutomatedA11yTesting.md]: As all of this was part of a comment, can we add a header (something like "Additional Tools") and just make the tools a list of links?
What do you think?

**Review by s-laugh — CHANGES_REQUESTED** [review]
Awesome!  Mostly just some formatting suggestions.

File: Testing/AutomatedA11yTesting.md (hunk 7 above)
Comment (j-paquette) [reply Testing/AutomatedA11yTesting.md]: Done.

Comment (s-laugh) [reply Testing/AutomatedA11yTesting.md]: Line 61 + 62 are part of the "Additional Tools" so should also be moved under that header.

**Review by s-laugh — DISMISSED** [review]

**Review by s-laugh — DISMISSED** [review]

**Review by s-laugh — APPROVED** [review]

**Pull request merged**